app/src/components/navigation/ProfileMenu.vue

```diff
@@ -61,21 +107,35 @@ const userNavigation: {
             leave-to-class="transform opacity-0 scale-95"
         >
             <MenuItems
-                class="absolute right-0 z-10 mt-2.5 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-zinc-900/5 focus:outline-none"
+                class="absolute right-0 z-10 mt-2.5 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-zinc-900/5 focus:outline-none dark:bg-zinc-800"
             >
                 <MenuItem v-for="item in userNavigation" :key="item.name" v-slot="{ active }">
                     <button
                         :class="[
-                            active ? 'bg-zinc-50' : '',
-                            'block w-full cursor-pointer px-3 py-1 text-left text-sm leading-6 text-zinc-900 ',
+                            active ? 'bg-zinc-50 dark:bg-zinc-800' : '',
+                            'flex w-full cursor-pointer items-center gap-2 px-3 py-1 text-left text-sm leading-6 text-zinc-900 dark:text-white dark:hover:bg-zinc-500',
                         ]"
                         @click="item.action"
                     >
-                        {{ item.name }}
+                        <component
+                            :is="item.icon"
+                            class="h-5 w-5 flex-shrink-0 text-zinc-500"
+                            aria-hidden="true"
+                        />
+                        <div class="flex flex-col gap-y-0">
+                            {{ item.name }}
+                            <span
+                                class="-mt-0 text-[12px] text-zinc-500 dark:text-white"
+                                v-if="item.language"
+                                >{{ item.language.name }}</span
+                            >
+                        </div>
```

Comment:
<img width="129" alt="image" src="https://github.com/user-attachments/assets/610dc002-0268-4f89-88b1-71c398417901">
The distance between the Settings and Theme icons are closer than between the other icons. This is due to the language being inserted underneath the "Language" menu item, but it makes the "Settings" menu item feel a bit too tight compared to the rest.

The menu item height could perhaps be made a bit bigger, so that it can fit the language name without expanding, and ensure that the spacing between the icons are visually equal?